index.html

```diff
@@ -46,5 +46,147 @@ <h3 class="title_2">LET’S CONNECT</h3>
+        <p class="para_2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button class="button" type="button" value="see project">see project</button>
+      </div>
+      <div class="work_1">
+        <img src="./image/Snapshoot Portfolio (3).png" alt="Snap_4" />
+        <h2 class="tonic">Multi-Post Stories</h2>
+        <ul class="details">
+          <li class="canopy">CANOPY</li>
+          <li class="dot"></li>
+          <li class="back_end">Back End Dev</li>
+          <li class="dot"></li>
+          <li class="a_2015">2015</li>
+        </ul>
+        <p class="para_2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button class="button" type="button" value="see project">see project</button>
+      </div>
+    </section>
+    <section class="about-section">
+      <h2 class="title_1">About <br />Myself</h2>
+      <p class="para_1">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="title_2">LET’S CONNECT</h3>
+      <ul class="social-icons">
+        <li>
+          <a href="https://twitter.com/MysticalAmissah"
+            ><img src="./image\twitter.png" alt="twitter"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/nicholas-amissah-153b09154"
+            ><img src="./image\Linkedin icon.png" alt="Linkedin"
+          /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./image\messenger.png" alt="messenger" /></a>
+        </li>
+        <li>
+          <a href="https://github.com/atok624"
+            ><img src="./image\github.png" alt="github"
+          /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./image\hands.png" alt="hands" /></a>
+        </li>
+      </ul>
+      <button class="button" type="button" value="Get my resume">Get my resume</button>
+    
+      <ul class="Lang">
+      <li class="languages">
+        <p class="Language_list">Languages</p><img class="lang_img" src="./image/down button.png" alt="">
+      </li>
+      <ul class="language_list">
+        <li class="lang_list"><img src="./image/javascipt.png" alt="Javascript"><h3>Javascript</h3></li>
+        <li class="lang_list"><img src="./image/css.png" alt="html"><h3>HTML</h3></li>
+        <li class="lang_list"><img src="./image/html.png" alt="css"><h3>CSS</h3></li>
```

Comment:
- [ ] Great Job implementing all the changes requested by the previous reviewer :muscle: I think I found why your image link is broken :smile: 
- I noticed that here your `h` in `html` started with a small case but that wasn't how the image was saved.
-  Your Image 
![image](https://user-images.githubusercontent.com/102699958/212487390-437c7aa7-2108-4b0d-9eda-d515bfbd1294.png)
- So please solve this and you should be well on your way to your next project :wink: